recipe/meta.yaml

```diff
@@ -1,19 +1,18 @@
 {% set name = "dedalus" %}
-{% set version = "2.2006" %}
-
+{% set version = "2.2207" %}
 {% set mpi_prefix = "mpi_" + mpi %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 7b6005c6c9b3b85c42da9a2ae85219b2722e2f61bda714d2518c4c058fd5d8df
+  url: https://files.pythonhosted.org/packages/6d/89/0f5b60cfd1d8dbe89f1f7293ee388ceef66f537ee8b5988cb6c7a8d35c6b/dedalus-2.2207.tar.gz
+  sha256: 158adebf4b4939bb3d322dc81176965997f84a9ce54e020210db52eda4773dc0
 
 build:
-  number: 4
-  skip: true  # [win or py>37]
+  number: 0
+  skip: true  # [win or py<35]
```

Comment:
```suggestion
  skip: true  # [win]
```

Reply:
I think the underlying package has a hard 3.5+ requirement for Python, but maybe we don't need that here?  I can see CF is only building for 3.7+, but I didn't know if that's a guarantee / documented somewhere.

Reply:
Yes, correct, we don't build for these old versions of Python anymore, so this version constraint is implicit anyhow.